src/dft/backends/mklgpu/mklgpu_helpers.hpp

```diff
@@ -82,95 +90,66 @@ inline constexpr oneapi::mkl::dft::config_param to_mklgpu(dft::detail::config_pa
                                          "Invalid config param.");
             return static_cast<oparam>(0);
     }
-}
+}*/
```

Comment:
Should it be removed instead of commented?

Reply:
Up to you. It's in the history, so it can always be restored.

Reply:
I think it is best to remove.